src/main/java/com/dumbdogdiner/stickyapi/common/translation/LocaleProvider.java

```diff
@@ -316,7 +316,7 @@ private void checkForLoadedLocale(@NotNull String name) throws IllegalArgumentEx
      * 
      * @return {@link java.util.TreeMap}
      */
-    public TreeMap<String, String> newVariables() {
+    public static TreeMap<String, String> newVariables() {
```

Comment:
Use HashMap please, it has a better complexity (O(1) v O(log n), otherwise known as infinitely less complex)

Reply:
I merged this from Zach's code, I'll fix it